models/topics.models.js

```diff
@@ -0,0 +1,18 @@
+const db = require("../db/connection");
+
+exports.selectTopics = () => {
+  return db.query("SELECT * FROM topics;").then((result) => {
+    return result.rows;
+  });
+};
+
+exports.selectArticlesById = (article_id) => {
```

Comment:
Also change the name of this function to represent what it is doing (selecting a single article)